adam/tests/config_manager_test.py

```diff
@@ -1,8 +1,9 @@
 from adam import ConfigManager
-import json
+import yaml
```

Comment:
super nit: could you alphasort the imports? Something like what PEP-8 suggests -- https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Fixed! N.b. [flake8-import-order](https://github.com/PyCQA/flake8-import-order) found another 142 issues in existing code that I'll leave for later.